exporters/pelorus/__init__.py

```diff
@@ -130,6 +133,9 @@ def upgrade_legacy_vars():
     github_user = utils.get_env_var("GITHUB_USER")
     git_username = utils.get_env_var("GIT_USER")
 
+    github_issue_label = utils.get_env_var("GITHUB_ISSUE_LABEL", 
+        DEFAULT_GITHUB_ISSUE_LABEL)
+
```

Comment:
There's no legacy var to upgrade here, no?

Reply:
correct.. thank you.. removed.